reference/src/layout/scalars.md

```diff
@@ -37,6 +37,12 @@ The `isize` and `usize` types are pointer-sized signed and unsigned integers.
 They have the same layout as the [pointer types] for which the pointee is
 `Sized`, and are layout compatible with C's `uintptr_t` and `intptr_t` types.
 
+> **Note**: C99 [7.18.2.4](https://port70.net/~nsz/c/c99/n1256.html#7.18.2.4)
+> requires `uintptr_t` and `intptr_t` to be at least 16-bit wide. That is, on
+> all platforms that we currently support, which have a C platform,
```

Comment:
Are you saying "if we support a platform and it has a C platform, then they are at least 16 bits wide", or "if we support a platform, then it has a C platform, and hence they are at least 16 bits wide"? Please clarify the wording.

Suggestion:
```
C99 [7.18.2.4](https://port70.net/~nsz/c/c99/n1256.html#7.18.2.4)
requires `uintptr_t` and `intptr_t` to be at least 16-bit wide.
All platforms we currently support have a C platform, and as a consequence,
`isize`/`usize` are at least 16-bit wide for all of them.
```

Reply:
Done